app/models/user.rb

```diff
@@ -58,6 +59,14 @@ def remove_access_to(organisations_to_remove)
     self.organisations = self.organisations - Array(organisations_to_remove)
   end
 
+  def associated_with_accredited_body?
+    providers
+      .where(recruitment_cycle: RecruitmentCycle.current)
```

Comment:
```suggestion
      .in_current_cycle
```
I found Graham's beautiful code
https://github.com/DFE-Digital/teacher-training-api/blob/a9e98c21cc7e214f383a96c8055da5ed533f6523/app/models/provider.rb#L91

Reply:
👍